makeflow/src/parser_jx.c

```diff
@@ -148,21 +137,21 @@ static int rule_from_jx(struct dag *d, struct jx *j) {
 	struct jx *inputs = jx_lookup(j, "inputs");
 	debug(D_MAKEFLOW_PARSER, "Parsing inputs");
 	if (!files_from_jx(n, 1, inputs)) {
-		debug(D_MAKEFLOW_PARSER, "Failure parsing inputs");
+		debug(D_MAKEFLOW_PARSER|D_NOTICE, "Failure parsing inputs");
```

Comment:
For this and the output one, `files_from_jx` will print a more descriptive error message and return failure, so this just needs to say if it was inputs or outputs